packages/pytest-simcore/src/pytest_simcore/environment_configs.py

```diff
@@ -35,12 +38,20 @@ def external_envfile_dict(request: pytest.FixtureRequest) -> EnvVarsDict:
     """
     envs = {}
     if envfile := request.config.getoption("--external-envfile"):
-        print("🚨 EXTERNAL `envfile` option detected. Loading", envfile, "...")
+        _logger.warning(
+            "🚨 EXTERNAL `envfile` option detected. Loading '%s' ...", envfile
+        )
 
         assert isinstance(envfile, Path)
         assert envfile.exists()
         assert envfile.is_file()
 
+        if not any(term in envfile.name.lower() for term in ("ignore", "secret")):
+            _logger.warning(
+                "🚨 CAUTION: The provided envfile '%s' might be pushed in this repository. Please rename it with `secret` or `ignore`",
+                envfile.name,
+            )
+
```

Comment:
Q: what does this mean?
Can't whatever is generated already contain a .ignore so that there is no risk of accidentally committing it. I would avoid such a risk.

Reply:
there nothing is generated here. It is a path passed by the developer into the  pytest CLI's option ``--external-envfile`.  I am just extra cautious here to avoid that secrets get leaked

Reply:
I rewrote the message